bin/convert-argv.js

```diff
@@ -122,7 +122,7 @@ module.exports = function(yargs, argv, convertOptions) {
 	function processConfiguredOptions(options) {
 		if(options === null || typeof options !== 'object') {
 			console.error('Config did not export an object or a function returning an object.');
-			process.exit(-1);
+			process.exit(-1); // eslint-disable-line
```

Comment:
I'm not sure it's good to do this. AFAIK you should use `process.exitCode` instead.